sigh/lib/sigh/download_all.rb

```diff
@@ -9,10 +9,16 @@ module Sigh
   class DownloadAll
     # Download all valid provisioning profiles
     def download_all(download_xcode_profiles: false)
-      UI.message("Starting login with user '#{Sigh.config[:username]}'")
-      Spaceship.login(Sigh.config[:username], nil)
-      Spaceship.select_team
-      UI.message("Successfully logged in")
+      if (token = api_token)
```

Comment:
Is this the equivalent to `if let variable = variable` in Swift? :epicwin: this is awesome haha

Reply:
@rogerluan Essentially!  I don’t like using it a lot but it works well here 😇